### ISSUE TITLE
Regression of the YouTube description page

### ISSUE THREAD

**Issue by demonff1** [TeamNewPipe/NewPipe#5697]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want

The previous layout of the descriptiom was better and similar to YouTube app. 

#### Is your feature request related to a problem? Please describe it

It is really annoying the description tab changed position. The previous location was best imo and it is a standard in other streaming service. 

#### Additional context

Frankly, it's an eyesore. It doesn't feel integrated to the interface.

**Comment by HTSamurai** [comment]
I agree. Probably a toggle to choose would be best, to have an option? We
don't want all the work to go to waste

**Comment by julian-fairfax** [comment]
I agree as well. It's something that bothered me in the new update (although the chapter support was more than welcome!)

**Comment by SameenAhnaf** [comment]
In my opinion, this is how everything should be accessible. 
Secondary toggle: Description
Scroll down: Comments
Swipe up bottom bar: Play queue

Complex options like text selection button should be gone. Short press on any link should open that link in browser and any other text should close description. Related to copying description: #7097
